backend/rumergy_backend/rumergy/serializers/data_log_serializer.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import serializers
+from rumergy_backend.rumergy.models import DataLog
+from django.contrib.auth.models import User
+
+class DataLogSerializer(serializers.ModelSerializer):
+    
+    # meter = serializers.PrimaryKeyRelatedField(many=True, queryset=Meter.objects.all())
+    user = serializers.PrimaryKeyRelatedField(many=False, queryset=User.objects.all())
+    # TODO: Import Meter model
+    
```

Comment:
Should data log measures be included here?

Reply:
Yeah, it should